client/database.go

```diff
@@ -5,67 +5,40 @@ import (
 	"fmt"
 
 	"github.com/rs/zerolog/log"
+	"github.com/tigrisdata/tigris-client-go/config"
 	"github.com/tigrisdata/tigris-client-go/driver"
 	"github.com/tigrisdata/tigris-client-go/schema"
 )
 
-// TxFunc is a user-provided function that will be run
-// within the context of a transaction.
-type TxFunc func(
-	ctx context.Context,
-	tr Tx,
-) (interface{}, error)
-
-// Tx is the interface for a client-level transaction. It does
-// not expose operations like Commit()/Abort as it is meant to be used
-// within the Transact() method which abstracts away those operations.
-type Tx interface {
-	driver.CRUDTx
-}
-
-// Database is the interface for interacting with a specific database
-// in Tigris.
+// Database is the interface for interacting with a Tigris database
 type Database interface {
-	// Transact runs the provided TransactionFunc in a transaction. If the
-	// function returns an error then the transaction will be aborted,
-	// otherwise it will be committed.
-	Transact(ctx context.Context, fn TxFunc) (interface{}, error)
+	CreateCollections(ctx context.Context, dbName string, model Model, models ...Model) error
+
+	Create(ctx context.Context) error
```

Comment:
What do you think about having "Get" method in the client and don't have anything here? Then if the database doesn't exist you'll create it. So once the database object is created by the user then it actually exists.